MediaGalleryUiApi/composer.json

```diff
@@ -0,0 +1,21 @@
+{
+    "name": "magento/module-media-gallery-ui-api",
+    "description": "Magento module responsible for the media gallery UI implementation API",
+    "require": {
+        "php": "~7.1.3||~7.2.0||~7.3.0",
+        "magento/framework": "*"
```

Comment:
Do I need to add `MediaGalleryUi` module in require?

Reply:
nope, it should be the other way around - MediaGalleryUi is supposed to implement interfaces of MediaGalleryUiApi - So MediaGalleryUiApi should be required for MediaGalleryUi module